packages/merge-styles/src/styleToClassName.test.ts

```diff
@@ -14,293 +11,348 @@ describe('styleToClassName', () => {
   });
 
   it('can register classes and avoid re-registering', () => {
-    let className = styleToClassName({ background: 'red' });
+    let className = styleToClassName({}, { background: 'red' });
 
     expect(className).toEqual('css-0');
     expect(_stylesheet.getRules()).toEqual('.css-0{background:red;}');
 
-    className = styleToClassName({ background: 'red' });
+    className = styleToClassName({}, { background: 'red' });
 
     expect(className).toEqual('css-0');
     expect(_stylesheet.getRules()).toEqual('.css-0{background:red;}');
 
-    className = styleToClassName({ background: 'green' });
+    className = styleToClassName({}, { background: 'green' });
 
     expect(className).toEqual('css-1');
     expect(_stylesheet.getRules()).toEqual('.css-0{background:red;}.css-1{background:green;}');
   });
 
   it('can have child selectors', () => {
-    styleToClassName({
-      selectors: {
-        '.foo': { background: 'red' }
+    styleToClassName(
+      {},
+      {
+        selectors: {
+          '.foo': { background: 'red' }
+        }
       }
-    });
+    );
 
     expect(_stylesheet.getRules()).toEqual('.css-0 .foo{background:red;}');
   });
 
   it('can have child selectors with comma', () => {
-    styleToClassName({
-      selectors: {
-        '.foo, .bar': { background: 'red' }
+    styleToClassName(
+      {},
```

Comment:
Is this just a different way to format the selectors (the { {}, {selectors : {....}}}?

Reply:
I added a new param to `styleToClassName`, and this is just calling it the same way it had been.